client/components/survey-container/components/question-step.tsx

```diff
@@ -49,7 +48,7 @@ const QuestionStep = ( {
 			goNext={ onSkip }
 			formattedHeader={
 				<FormattedHeader
-					align="left"
+					align="center"
```

Comment:
It could become an optional component prop, as the Entrepreneur flow expects a left-aligned header. If you agree on the solution, I can also help with a follow-up fix for that.

Reply:
I went ahead and implemented the fix, adding it here for reference: https://github.com/Automattic/wp-calypso/pull/91025

Reply:
Yes that sounds perfect